### DISCUSSION TITLE
Add support for cntrlpanel

### DISCUSSION THREAD

**Discussion by WilliamThe4rd** [pelican-dev/panel#15] [Ideas]
Cntrlpanel is a panel for ptero and will it support pelican?
https://github.com/Ctrlpanel-gg/panel

**Comment by Saturn745** [comment]
That seems to be a fork of Pterodactyl so that's something you would most likely need to take to the maintainers of Ctrlpanel.

**Reply by purejosh** [reply]
ctrlpanel is **not** a fork of Pterodactyl. It is a dedicated billing panel that uses ptero's API to allow for billing and deploying servers from its web interface.

**Reply by Saturn745** [reply]
I see, my apologizes

**Comment by purejosh** [comment]
Rename to: 
```
Native ctrlpanel support
```

**Comment by lancepioch** [comment]
Thanks for the suggestion, we are considering a first party modification or plugin along these lines! It will not be related to the project you linked.
